ckanext/canada/templates/public/organization/snippets/organization_search.html

```diff
@@ -0,0 +1,26 @@
+<form class="form-inline mrgn-bttm-sm mrgn-tp-md" role="form" method="get" data-module="select-switch">
+  <div class="input-group col-md-12">
+    <input type="text" class="form-control" id="field-organization-search" autocomplete="off" name="q"
+           value="{{ q }}" placeholder="{{ _('Search organizations...') }}">
```

Comment:
is the built-in org search going to do stemming properly to find french org names?

Reply:
In normal circumstances, no. BUT we are still using our `canada_org_title_translated_save` validator on the org title field.

That is the one which takes the fluent english and french title values and saves it as a piped string in the org's core title field.

The group list action with the `q` arg does `%{q}%` on the org name, title, and description. So yeah it will work with the French title. Shall I add a comment in the validator to never remove it as it is very important?

Reply:
Instead of "never remove this" how about a list of the things that depend on it, like "org page search, recombinant csv output, ... (there may be others)"